<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,18,5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,7">
      <item>
       <widget class="QLabel" name="lbl_titulo">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cadastrar fabricação</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,4,1">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_data">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Data</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="date_data">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_rendimento">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Rendimento</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,3">
            <item>
             <widget class="QLineEdit" name="txt_rendimento">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_unidade_rendimento">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_tempo">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tempo gasto</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,3">
            <item>
             <widget class="QSpinBox" name="spin_tempo">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_minutos">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_iniciar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Iniciar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="14,9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
            <item>
             <widget class="QLabel" name="lbl_receita">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Receita</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_receita">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,12">
          <item>
           <widget class="QLabel" name="lbl_ingredientes">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ingredientes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tb_ingredientes">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="rowCount">
             <number>3</number>
            </property>
            <property name="columnCount">
             <number>7</number>
            </property>
            <row/>
            <row/>
            <row/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7,4,2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
            <item>
             <widget class="QLabel" name="lbl_ingrediente">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ingrediente</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_ingrediente">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cancelar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tb_embalagens">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="6,8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="4,8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,5">
          <item>
           <widget class="QLabel" name="lbl_custo_final">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Custo final</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_custo_final">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_cad_limpar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cadastrar e limpar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cad_sair">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cadastrar e sair</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_recomecar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Recomeçar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_voltar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>date_data</tabstop>
  <tabstop>txt_rendimento</tabstop>
  <tabstop>txt_unidade_rendimento</tabstop>
  <tabstop>spin_tempo</tabstop>
  <tabstop>btn_iniciar</tabstop>
  <tabstop>combo_receita</tabstop>
  <tabstop>tb_ingredientes</tabstop>
  <tabstop>txt_ingrediente</tabstop>
  <tabstop>btn_cancelar</tabstop>
  <tabstop>tb_embalagens</tabstop>
  <tabstop>txt_custo_final</tabstop>
  <tabstop>btn_cad_limpar</tabstop>
  <tabstop>btn_cad_sair</tabstop>
  <tabstop>btn_recomecar</tabstop>
  <tabstop>btn_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
